### PR TITLE
[TinyMCE] Fix windowManager attribute name in Editor type

### PR DESCRIPTION
**Pull request by thiagolacerda**
Since it is a property of the Editor, it should follow the standard
followed by other properties: first letter should be lowercase

Signed-off-by: Thiago Lacerda <thiagotbl@gmail.com>

### CODE REVIEW

**Comment by iPoul** [comment]
👍

**Review by iampoul — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR appears ready to merge pending express review by a maintainer.

**Pull request merged**
